integration-tests/bats/blame.bats

```diff
@@ -50,55 +50,53 @@ SQL
     [[ "$output" =~ "usage" ]] || false
 }
 
+@test "blame: too many arguments shows usage" {
+    run dolt blame HEAD blame_test foo
+    [ "$status" -eq 1 ]
+    [[ "$output" =~ "blame has too many positional arguments" ]] || false
+}
+
 @test "blame: annotates a small table with simple history" {
     # should be the same as dolt blame HEAD blame_test
     run dolt blame blame_test
     [ "$status" -eq 0 ]
 
-    # TODO: Make these assertions better
-    [[ "$output" =~ "Thomas Foolery" ]] || false
-    [[ "$output" =~ "create blame_test table" ]] || false
+    [[ "$output" =~ "1".*"Thomas Foolery".*"create blame_test table" ]] || false
     [[ ! "$output" =~ "Richard Tracy" ]] || false
     [[ ! "$output" =~ "add richard to blame_test" ]] || false
-    [[ "$output" =~ "Harry Wombat" ]] || false
-    [[ "$output" =~ "replace richard" ]] || false
-    [[ "$output" =~ "Johnny Moolah" ]] || false
-    [[ "$output" =~ "add more people" ]] || false
+    [[ "$output" =~ "2".*"Harry Wombat".*"replace richard" ]] || false
+    [[ "$output" =~ "3".*"Johnny Moolah".*"add more people" ]] || false
+    [[ "$output" =~ "4".*"Johnny Moolah".*"add more people" ]] || false
 }
 
 @test "blame: blames HEAD when commit ref omitted" {
     run dolt blame blame_test
     [ "$status" -eq 0 ]
-    [[ "$output" =~ "Thomas Foolery" ]] || false
-    [[ "$output" =~ "create blame_test table" ]] || false
+
+    [[ "$output" =~ "1".*"Thomas Foolery".*"create blame_test table" ]] || false
     [[ ! "$output" =~ "Richard Tracy" ]] || false
     [[ ! "$output" =~ "add richard to blame_test" ]] || false
-    [[ "$output" =~ "Harry Wombat" ]] || false
-    [[ "$output" =~ "replace richard" ]] || false
-    [[ "$output" =~ "Johnny Moolah" ]] || false
-    [[ "$output" =~ "add more people" ]] || false
+    [[ "$output" =~ "2".*"Harry Wombat".*"replace richard" ]] || false
+    [[ "$output" =~ "3".*"Johnny Moolah".*"add more people" ]] || false
+    [[ "$output" =~ "4".*"Johnny Moolah".*"add more people" ]] || false
 }
 
 @test "blame: works with HEAD as the commit ref" {
-    skip "SQL views do no support AS OF queries"
-
     run dolt blame HEAD blame_test
     [ "$status" -eq 0 ]
-    [[ "$output" =~ "Thomas Foolery" ]] || false
-    [[ "$output" =~ "create blame_test table" ]] || false
+
+    [[ "$output" =~ "1".*"Thomas Foolery".*"create blame_test table" ]] || false
```

Comment:
Not clear what this does and why. Can you educate me?

Reply:
`*` is a wildcard that matches any number of any character. My intention was to check that the correct phrases are on the correct lines associated with their `pk`. I see a later test (L147) that uses `lines` to do a similar thing in a cleaner way so I may switch over to that.

Reply:
Got it. I had to take a look at blame's output to understand what you were doing.

The unfortunate thing here is that there are a lot of columns of output we are just ignoring. So long as other tests cover those I think this makes sense. Can you verify?

Reply:
Got it. I had to take a look at blame's output to understand what you were doing.

The unfortunate thing here is that there are a lot of columns of output we are just ignoring. So long as other tests cover those I think this makes sense. Can you verify?

Reply:
I can add the email column in these checks as well,, the commit date and commit hash might be harder to check since those will change every time the test is run I think. Do you think that suffices to check for accuracy?

Reply:
Just read your comment below. I'll add a check for the headers if the table and see if I can test that the commit date and commit hash columns aren't empty at least.